pkgs/development/compilers/rust/rustfmt.nix

```diff
@@ -15,6 +15,11 @@ rustPlatform.buildRustPackage rec {
     rustPlatform.rust.rustc.llvm
   ] ++ lib.optional stdenv.isDarwin Security;
 
+  preFixup = lib.optionalString stdenv.isDarwin ''
```

Comment:
Might be worth adding my comment from Clippy's derivation here, or just reference it.

Reply:
I agree that it should be commented. I was unsure on what was the best way to do so. I've copied the comment verbatim.